ocm_python_wrapper/cluster.py

```diff
@@ -256,33 +257,85 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    def filter_required_addon_parameters(self, parameters=None):
+        """Filters out addon parameter by specified cluster-related conditions."""
+
+        filtered_parameters = []
+        if parameters is None:
+            parameters = []
+
```

Comment:
please split.
get a dict of required parameters for cluster resource that meet cluster config
{param_name:  {default_value: ...,}}

then iterate over the dict
- if param does not exist in user params and has  default value - update addon params installation, else - append to missing list

Reply:
Updated - can I update the logic with better approach?